cvat/apps/quality_control/quality_reports.py

```diff
@@ -2394,6 +2401,15 @@ def _compute_task_report(
                 task_frame_results_counts[frame_id] = 1 + frame_results_count
                 task_frame_results[frame_id] = task_frame_result
 
+        if confusion_matrices:
+            confusion_matrix = np.sum(confusion_matrices, axis=0)
```

Comment:
As we discovered in the parallel [PR](https://github.com/opencv/cvat/pull/7619), this method is less efficient now that simple accumulation during iteration. Could you please update the code?